pcsx2/GS/GS.cpp

```diff
@@ -622,7 +622,7 @@ void GSGetAdaptersAndFullscreenModes(
 				if (adapters)
 					*adapters = D3D::GetAdapterNames(factory.get());
 				if (fullscreen_modes)
-					*fullscreen_modes = D3D::GetFullscreenModes(factory.get(), EmuConfig.GS.Adapter);
```

Comment:
Whoops, this one should also be EmuConfig, since it gets called from the UI thread.